causing/graph.py

```diff
@@ -157,13 +158,21 @@ def graph_to_dot(
     )
 
     for node, data in g.nodes(data=True):
-        eff_str = utils.fmt_min_sig(data["effect"], min_sig_figures, percent=in_percent)
+        eff_str = utils.fmt_min_sig(
+            data["effect"] if abs(data["effect"]) > cutoff else 0,
+            min_sig_figures,
+            percent=in_percent,
+        )
```

Comment:
This and the same code below look quite redundant, but may it's not really worth de-duplicating it.